src/main/java/de/tum/in/www1/artemis/service/ldap/LdapUserDto.java

```diff
@@ -12,13 +12,14 @@
 
 @Entry(base = "ou=users", objectClasses = { "imdPerson" })
 @Profile("ldap | ldap-only")
+// TODO: double check if we can use a Record here
 public final class LdapUserDto {
```

Comment:
Is this a follow-up TODO or a forgotten one?

Reply:
a follow-up TODO